### PR TITLE
Utility to print metadata table

### PR DESCRIPTION
**Pull request by milleruntime**
I thought that a utility to easily print different things from the new Ample classes would be useful. I am not sure what to print though. Most of the tablet stuff is available through other utilities like `listtablets` and could just be obtained through a scan of the metadata table. My thinking was something more user friendly than having to scan the metadata table, and deciphering the schema.

### CODE REVIEW

**Review by dlmarion — COMMENTED** [review]
This looks ok to me. My only question is whether or not the endRow and prevEndRow of the extent should be obscured or not. I know that this is done in the Monitor. It also looks like it is done in GetSplitsCommand. I'm not sure that I am familiar with the rules on when this should be done vs not done.

**Comment by milleruntime** [comment]
> This looks ok to me. My only question is whether or not the endRow and prevEndRow of the extent should be obscured or not. I know that this is done in the Monitor. It also looks like it is done in GetSplitsCommand. I'm not sure that I am familiar with the rules on when this should be done vs not done.

Yeah we could. I did not obscure it in the `listtablets` command though. But that was kinda the whole point of that command.

File: server/base/src/test/java/org/apache/accumulo/server/util/PrintMetadataTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.apache.accumulo.server.ServerContext;
+import org.junit.jupiter.api.Test;
+
+public class PrintMetadataTest {
+  @Test
+  public void test() throws Exception {
+    ServerContext serverContext = new MockServerContext().withTables("table1").withAmple().done();
+
+    PrintMetadata printMetadata = new PrintMetadata(serverContext);
+    printMetadata.execute(new String[] {});
```
Comment (dlmarion) [inline server/base/src/test/java/org/apache/accumulo/server/util/PrintMetadataTest.java:31]: Should this check the output?

File: core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java [hunk 2]
```diff
@@ -471,4 +471,9 @@ private static Optional<TServerInstance> checkServer(ClientContext context, Stri
     }
     return server;
   }
+
+  @Override
+  public String toString() {
+    return getExtent() + " " + getDirName() + " " + getLocation();
```
Comment (dlmarion) [inline core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java:477]: I would remove this, or move this if it's being used in the new code. TabletMetadata has so many fields and this is just printing a few. If someone modifies this method later, and it exposes data that we don't want exposed in the PrintMetadata utility, then that could be bad.

File: server/base/src/test/java/org/apache/accumulo/server/util/PrintMetadataTest.java (hunk 1 above)
Comment (milleruntime) [reply server/base/src/test/java/org/apache/accumulo/server/util/PrintMetadataTest.java:31]: This is incomplete...

**Comment by milleruntime** [comment]
@dlmarion I am tempted to just close this PR because I can't think of what exactly to print that isn't available with some other tool.

**Comment by dlmarion** [comment]
I mean, if you think it's usefulness is questionable, then I agree. No reason to create something that needs to be maintained if there is not utility in it.

**Pull request closed without merging**